### ISSUE TITLE
Updating a TXT field by advertising an existing target does not work

### ISSUE THREAD

**Issue by stephanemagnenat** [aseba-community/aseba#692]
Commit 0012396f adds support for the busy flag in dummynode, that should be set to true (and thus present in the TXT flag) if the target is being used.

While the call `Zeroconf::advertise()` is called with the updated `TxtRecord`, the description received by the client (for instance `asebatargetlist-thread`) is not updated.

**Comment by stephanemagnenat** [comment]
This can easily be seen by launching a dummy and observing it, for example on macOS with:
```
dns-sd -B _aseba._tcp .
dns-sd -L "Dummy Node 0" _aseba._tcp .
```

**Comment by davidjsherman** [comment]
The unit tests in [branch zeroconf-unit-tests](https://github.com/davidjsherman/aseba/tree/zeroconf-unit-tests) need to be finished (and updated wrt the new Zeroconf interface?).

**Comment by stephanemagnenat** [comment]
Fixed by commit cc0c5112.
